packages/cxl-ui/scss/cxl-institute-content.scss

```diff
@@ -0,0 +1,65 @@
+@import "./utils/mq";
+
+:host {
+  background: #fff;
+  border-radius: 0;
+  bottom: 0;
+  box-shadow: -4px 1px 15px -8px var(--lumo-shade-50pct);
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  left: 0;
+  overflow: hidden;
+  position: absolute;
+  right: 0;
+  top: 0;
+
+  // For sidebar toggle
+  transition: all 0.3s cubic-bezier(0.165, 0.84, 0.44, 1);
+
+  @include mq($from: tablet) {
+    height: 100vh;
+  }
+
+  @include mq($from: tablet) {
+    border-bottom-left-radius: var(--lumo-border-radius-l);
+    border-top-left-radius: var(--lumo-border-radius-l);
+  }
+}
+
+:host([hidesidebar="true"]) {
+  border-bottom-left-radius: 0;
+  border-top-left-radius: 0;
+  left: -315px;
+}
+
+:host([hidesidebar="true"][sidebarsize="small"]) {
+  border-bottom-left-radius: 0;
+  border-top-left-radius: 0;
+  left: -250px;
+}
+
+.wrap {
+  height: auto;
+  max-width: none;
+  overflow: auto;
+  padding: 0;
+  width: 100%;
+}
+
+[part="content"] {
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: column;
+  overflow: auto;
+  padding: var(--lumo-space-m) var(--lumo-space-m);
+
+  &[media-size="wide"] {
+    margin: 0 auto;
+    max-width: 35em;
```

Comment:
This should probably be derived from a breakpoint value `calc()`.

Reply:
I was wrong, but I will move this into a variable at the top, so it's not a totally obscure thing.